sessions/skv/skv.go

```diff
@@ -0,0 +1,266 @@
+// Package memory provides the default in-memory session store with skv backing the memory stuff up
```

Comment:
This file looks good apart from the documentation ("// Package memory ..."), new session driver options are always welcome!